### PR TITLE
Add functionality to add and remove task milestone.

### PR DESCRIPTION
**Pull request by Nowembabazi**
##In this project I achieved the following;
-I added functionality to this project.
-Used webpack for this project.
-used index.html for the project.
-[Add functionality to add and remove task]
-Linters are working perfectly.

### CODE REVIEW

**Review by Danieldotcomcoder — COMMENTED** [review]
Hi @Nowembabazi , 


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't include a professional Readme File , Kindly refer to this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍 👍  to add a good Readme File filled with information about the project 😄 😄 then submit again, Thank you 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
-  },
-  {
-    index: 4,
-    description: "I love microverse",
-    completed: false,
-  },
-];
+import addedTask from "./module/class.js"
+import { addToLocalStorage, getFromLocalStorage } from "./module/localstorage.js"
+import displayList from "./module/displaylist.js"
```
Comment (mikethreels) [inline src/index.js:6]: - [ ] in these lines you are importing from the module folder however the folder doesn't exist it looks like you might have forgotten to push these changes, therefore the user receives below error.

afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Nickson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/class.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import {addToLocalStorage} from "./localstorage.js"
+
+class Task {
```
Comment (OLIPLICHE) [inline src/modules/class.js:3]: - [ ] Please you have issues with your codes. Kindly fix it. 
- [ ] Make sure all the elements of the user interface will be fully functional according to the Project requirements.
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Nowembabazi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**